beacon-chain/sync/validate_beacon_blocks.go

```diff
@@ -179,16 +179,22 @@ func (s *Service) setSeenBlockIndexSlot(slot uint64, proposerIdx uint64) {
 
 // Returns true if the block is marked as a bad block.
 func (s *Service) hasBadBlock(root [32]byte) bool {
+	if true {
+		return false
+	}
```

Comment:
If we are not marking blocks as bad due to context error, should we remove above noop?

Reply:
yep, accidentally pushed that!